### PR TITLE
Add method option to the IFileUploadConfigFile interface

### PR DESCRIPTION
**Pull request by Airboy**
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: [Upload service > options](https://github.com/danialfarid/ng-file-upload#upload-service)

### CODE REVIEW

File: types/ng-file-upload/index.d.ts [hunk 1]
```diff
@@ -268,6 +268,10 @@ declare module 'angular' {
              * @type {string}
              */
             arrayKey?: string;
+            /**
+             * Add which HTTP method to use: 'POST' or 'PUT'(html5)
```
Comment (thewarpaint) [inline types/ng-file-upload/index.d.ts]: Please add a space in between: `'PUT' (HTML5)`.

Comment (Airboy) [reply types/ng-file-upload/index.d.ts]: Nice catch, fixed.

Comment (thewarpaint) [reply types/ng-file-upload/index.d.ts]: I still see it the way it was before :(

Comment (Airboy) [reply types/ng-file-upload/index.d.ts]: Fixed. My bad, sorry ;-)

**Review by thewarpaint — APPROVED** [review]

File: types/ng-file-upload/index.d.ts (hunk 1 above)
Comment (thewarpaint) [reply types/ng-file-upload/index.d.ts]: No worries. Thanks for fixing!

**Pull request merged**